### ISSUE TITLE
put testing pathtable that is in code into a csv

### ISSUE THREAD

**Issue by JoeZiminski** [neuroinformatics-unit/datashuttle#98]
Would it be easier to store this in a csv or something? It's hard to parse as code.

_Originally posted by @adamltyson in https://github.com/neuroinformatics-unit/datashuttle/pull/69#discussion_r1176292303_

**Comment by JoeZiminski** [comment]
Hey @adamltyson this table has an f-string that takes a base dir (depending on where the tests are run), could replicate but saving everything but this column as csv, then loading this column in and stitch it together with the strings. Also I believe the bool type would have to be converted from string? In which case I think current (more ugly) format is easier to work with

**Comment by adamltyson** [comment]
Sounds good. Just wondered.
